Interoperability/python/qsharp/__init__.py

```diff
@@ -47,27 +49,24 @@
     SIMULATOR_COLOR = ca.Fore.BLUE + ca.Back.WHITE
     RESET_COLOR = ca.Style.RESET_ALL
 except:
-    ca = None
+    CA = None
     SIMULATOR_COLOR = ""
     RESET_COLOR = ""
 
 try:
     import qutip as qt
 except:
-    qt = None
+    QT = None
```

Comment:
Intended ?

Reply:
Same as above, QuTiP isn't used for all features, so guarding in this way lets us generate more sensible error messages when QuTiP isn't present.